templates/zerver/help/make-a-user-an-administrator.md

```diff
@@ -0,0 +1,29 @@
+# Make a user an administrator
+
+> With great power comes great responsibility.
+
+By default, everyone in a realm is a user; this limits them from modifying realm-wide settings, such as changing the organization name, activating/deactivating users, deleting streams, etc. Zulip realm administrators can give a user administrative rights. To do so, follow the steps below:
+
+1. Click the (![cog](/static/images/help/cog.png)) in the upper right corner of the right sidebar.
+
+2. Select **Administration** from the dropdown menu.
+
+    ![admin](/static/images/help/admin.png)
+
+3. Upon clicking **Administration**, your view will be replaced with the **Administration** page. Click on **Users** in the tabs list.
+
+    ![admin panel](/static/images/help/admin-panel-generic.png)
+
+4. In the **Users** section, you will be presented with various settings over your users. Click on **Make admin** for the user(s) that you wanted to make an administrator.
```

Comment:
1. I'd say "settings for your users".

2. In the second sentence I'd use both verbs in present tense, that would change "wanted" to "want". Not sure if "wish" wouldn't be better here, or is that too formal?